EventsExpress/ClientApp/src/components/user-info/user-block.js

```diff
@@ -7,11 +7,9 @@ import IconButton from "@material-ui/core/IconButton";
 export function UserBlock(props) {
     const { user, isCurrentUser, block, unblock} = props;
 
-    function onBlock(){props.openDialog({title: "Block user",message: "Are u sure?",action: props.block})}
-
     return (isCurrentUser) ? <td> </td> : <> <td className="align-middle">
                 <div className="d-flex justify-content-center align-items-center">
-                {(user.isBlocked == true)
+                {(user.isBlocked === true)
```

Comment:
No need to compare bool value to true. Just use the value itself.

Reply:
```
if (user.isBlocked.toString() === "True" ||
    user.isBlocked.toString() === "true" ||
    user.isBlocked.toString() === "TRUE" ||
) { just kidding }
```